### PR TITLE
Unit test on point search possibility

### PR DESCRIPTION
**Pull request by 47souls**
Finally provided unit test on point search possibility. It has 3 methods:
1) Search place by name, if place with this name exist.
2) Search place by name, if place with this name doesn`t exist.
3) Search place by empty name.

### CODE REVIEW

File: src/main/java/ua/softserve/rv_018/greentourism/service/PlaceService.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ua.softserve.rv_018.greentourism.service;
+
+import ua.softserve.rv_018.greentourism.model.Place;
+
+public interface PlaceService {
+
+	Place findPlaceByName(String name);
+
+}
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/service/PlaceService.java:9]: What is that?
Are you going to put this to the dev?

File: src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+public class PlaceServiceImpl implements PlaceService {
+
+	@Override
+	public Place findPlaceByName(String name) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+}
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java:13]: What is that?
Are you going to put this to the dev?
Where is implementation?

Comment (47souls) [inline src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java:13]: No I just added this stub to provide posibility of testing. I"m gonna push this stub to  branch of my functionality and in other stub i"ll implement this method.

**Comment by Sabfir** [comment]
reviewed

File: src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java (hunk 2 above)
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java:13]: got it, my fault

**Pull request merged**